Travis failure.

@sbrunner @pgiraud Since yesterday Travis is failling. 
Has something been changed and do we need to modify something ? 

```
Traceback (most recent call last):
  File ".build/venv/bin/c2c-template", line 9, in <module>
    load_entry_point('c2c.template==1.2.0.dev2', 'console_scripts', 'c2c-template')()
  File "/home/travis/build/Geoportail-Luxembourg/geoportailv3/.build/venv/local/lib/python2.7/site-packages/c2c/template.py", line 185, in main
    _proceed(files, used_vars, options)
  File "/home/travis/build/Geoportail-Luxembourg/geoportailv3/.build/venv/local/lib/python2.7/site-packages/c2c/template.py", line 209, in _proceed
    bottle_template(files, used_vars, engine)
  File "/home/travis/build/Geoportail-Luxembourg/geoportailv3/.build/venv/local/lib/python2.7/site-packages/c2c/template.py", line 240, in bottle_template
    template, **used_vars
  File "/home/travis/build/Geoportail-Luxembourg/geoportailv3/.build/venv/bin/bottle.py", line 3595, in template
    return TEMPLATES[tplid].render(kwargs)
  File "/home/travis/build/Geoportail-Luxembourg/geoportailv3/.build/venv/bin/bottle.py", line 3280, in render
    return self.tpl.render(**_defaults)
  File "/home/travis/build/Geoportail-Luxembourg/geoportailv3/.build/venv/local/lib/python2.7/site-packages/mako/template.py", line 443, in render
    return runtime._render(self, self.callable_, args, data)
  File "/home/travis/build/Geoportail-Luxembourg/geoportailv3/.build/venv/local/lib/python2.7/site-packages/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/home/travis/build/Geoportail-Luxembourg/geoportailv3/.build/venv/local/lib/python2.7/site-packages/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/home/travis/build/Geoportail-Luxembourg/geoportailv3/.build/venv/local/lib/python2.7/site-packages/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "node_modules_ngeo__tx_config_mako", line 21, in render_body
AttributeError: 'Undefined' object has no attribute 'strip'
make: *** [node_modules/ngeo/.tx/config] Error 1
```

--- a/geoportailv3.mk
+++ b/geoportailv3.mk
@@ -5,7 +5,7 @@ VARS_FILE = vars_geoportailv3.yaml
 VARS_FILES += ${VARS_FILE}
 endif
 
-TEMPLATE_EXCLUDE += LUX_alembic/script.py.mako
+TEMPLATE_EXCLUDE += LUX_alembic/script.py.mako node_modules
 
 LANGUAGES = en fr de lb
 CGXP = FALSE

